### PR TITLE
Introduce FIP for built-in actors API

### PR DESCRIPTION
**Pull request by arajasek**
Corresponds to Discussion #401 

Please find the prototyped implementation [here](https://github.com/filecoin-project/builtin-actors/tree/integration/builtin-api).

### CODE REVIEW

File: FIPS/fip-actors-api.md [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+
+This is not a particularly security-sensitive FIP. No new functionality is being introduced that can be used to attack the network, and bugs in the implementation will only lead to incorrect behaviour being observed by user actors.
+
+## Incentive Considerations
+<!--All FIPs must contain a section that discusses the incentive implications/considerations relative to the proposed change. Include information that might be important for incentive discussion. A discussion on how the proposed change will incentivize reliable and useful storage is required. FIP submissions missing the "Incentive Considerations" section will be rejected. An FIP cannot proceed to status "Final" without a Incentive Considerations discussion deemed sufficient by the reviewers.-->
+
+This FIP is intended to expose core information and functionality to user-defined actors. In doing so, it is hoped that this FIP will support the development of novel actors that interact with Filecoin's storage mechanics and unlock new use cases.
```
Comment (anorth) [inline FIPS/fip-actors-api.md:240]: I think this was meant to go in Product Considerations instead

**Review by anorth — APPROVED** [review]
Great work

**Comment by kaitlin-beegle** [comment]
Thanks, @arajasek! Going to go ahead and merge so we can number the FIP.

**Review by raulk — COMMENTED** [review]
Any reason why this does not specify full interfaces/signatures, just method names? I think it’s incomplete without them.

**Pull request merged**